OpeningTimes.js

```diff
@@ -96,9 +81,33 @@ class OpeningTimes {
     return this._openingTimes[this._getDayName(moment)];
   }
 
-  _getOpenSessions(moment) {
+  _getOpeningTimesSessionForMoment(moment, daysLookAhead) {
+    // Get sessions for week (including yesterday for the cases where opening hours span midnight)
+    let returnValue;
+    for (let day = daysLookAhead - 1; day >= -1; day--) {
```

Comment:
Given this function i.e. `_getOpeningTimesSessionForMoment` is called once the `daysLookAhead` is set to `1` could this `for()` be removed?